pkg/controller/podautoscaler/horizontal.go

```diff
@@ -615,20 +615,13 @@ func (a *HorizontalController) reconcileAutoscaler(ctx context.Context, hpaShare
 		metricName            string
 	)
 
-	desiredReplicas := int32(0)
 	rescaleReason := ""
-
-	var minReplicas int32
-
+	var desiredReplicas, minReplicas int32 = 0, 1
```

Comment:
IMO `var minReplicas int32 = 1` right before the `hpa.Spec.MinReplicas != nil` if makes the code more readable 👍

IMO multiple variable initialization makes the code less readable. Please add `var desiredReplicas int32` right before `scale.Spec.Replicas == 0 && minReplicas != 0` if instead.

Reply:
> IMO `var minReplicas int32 = 1` right before the `hpa.Spec.MinReplicas != nil` if makes the code more readable 👍
> 
> IMO multiple variable initialization makes the code less readable. Please add `var desiredReplicas int32` right before `scale.Spec.Replicas == 0 && minReplicas != 0` if instead.

done.